### DISCUSSION TITLE
How about close all existing PRs?

### DISCUSSION THREAD

**Discussion by Xuanwo** [tikv/client-rust#406] [General]
Hi community! I wanted to let you know that client-rust has been migrated from `grpc-sys` to `tonic`. As a result, our existing pull requests are more difficult to review and unlikely to be merged. Would it be possible for us to close all of the current PRs and notify their authors so we can start fresh?

cc @andylokandy @ekexium @pingyu and all @tikv/maintainers

**Comment by pingyu** [comment]
+1
